cpp/util/etcd.cc

```diff
@@ -980,7 +980,8 @@ list<EtcdClient::HostPortPair> SplitHosts(const string& hosts_string) {
   list<EtcdClient::HostPortPair> ret;
   for (const auto& h : hosts) {
     vector<string> hp(util::split(h, ':'));
-    CHECK_EQ(2, hp.size()) << "Invalid host:port string: '" << h << "'";
+    CHECK_EQ(static_cast<size_t>(2), hp.size()) << "Invalid host:port string: '"
```

Comment:
I didn't want to assume the length of a size_t so I've made everywhere cast to the same type.